solutions/dashboard/deployment/dashboard/portscannercrd.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
@shailesh-vaidya, are these changes done by you?

Reply:
Yes